server/models/defaultValues.js

```diff
@@ -0,0 +1,29 @@
+
+import pool from "../configurations/db-config"
+
+const inserter = `
+  INSERT INTO users(
+    firstName,lastName,email,password,address,bio,occupation,expertise,is_mentor,is_admin
+  )VALUES(
+    'IRANZI',
```

Comment:
Why do you have this dummy data?

Reply:
For running tests!

Reply:
You need to create a test db, So this should be inserted in the test db